### PR TITLE
Create header component 

### LINKED ISSUE #66: Create header
You need to create a header component for the project.
1) It should have logo and existing options ->  Logo, Home , about, Contributors Contact and SignIn option.
2) For logo place a dummy image and display text beside it.
3) On hover text should be underlined. (Use color code from the image.)
4) SignIn option should have a user icon, onclick a dropdown menu.
5) Mobile view of the header should look same as current view.
*Prefer using flex-box*
header

Everything should be properly aligned and responsive. Use styled components and bootstrap.

### PR DESCRIPTION
**Pull request by nikita-1801**
# Description

Created a responsive header component as directed in the issue.

Fixes #66 

## Type of change

- [x] Frontend

## Checklist

- [ ] I have named my files and folder, according to this project's guidelines.
- [x] My code follows the style guidelines of this project.
- [x] My Pull Request has a descriptive title. (not a vague title like `Update index.md`)
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] My code additions/changes do not produce any warnings.

## Resultant Style

image

For logo, I have added a fontawesome icon and added the code for logo image as a comment.

### CODE REVIEW

File: src/components/layout/Navbar.jsx [hunk 1]
```diff
@@ -1,62 +1,51 @@
 import React from "react";
 import { NavLink } from "react-router-dom";
-import { FaPaperPlane } from "react-icons/fa";
-
```
Comment (ashishnagpal2498) [inline src/components/layout/Navbar.jsx:4]: import image here.

File: src/components/layout/Navbar.jsx [hunk 2]
```diff
@@ -1,62 +1,51 @@
...
-                    /*--------For small screen----------- */
-                    <NavLink key={i+1} exact className="d-lg-none d-xl-none nav-link my-2" to={i.path}>
-                    {i.name}
-                    </NavLink>
-                ])
-              }
-            </div>
+      <nav class="navbar navbar-expand-lg navbar-light">
+      <a class="navbar-brand">
+        <i class="fab fa-stripe-s logo" ></i>
```
Comment (ashishnagpal2498) [inline src/components/layout/Navbar.jsx:17]: Add the logo here.

File: src/components/layout/Navbar.jsx [hunk 3]
```diff
@@ -1,62 +1,51 @@
...
-
-                    /*--------For small screen----------- */
-                    <NavLink key={i+1} exact className="d-lg-none d-xl-none nav-link my-2" to={i.path}>
-                    {i.name}
-                    </NavLink>
-                ])
-              }
-            </div>
+      <nav class="navbar navbar-expand-lg navbar-light">
+      <a class="navbar-brand">
```
Comment (ashishnagpal2498) [inline src/components/layout/Navbar.jsx:16]: Make this back to NavLink

File: src/components/layout/Navbar.jsx [hunk 4]
```diff
@@ -1,62 +1,51 @@
...
+        {/*For logo: <img class="logo" src="" width="" height="" alt="logo"/> */}
+        Social Scheduler</a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+    <div class="collapse navbar-collapse" id="navbarNavDropdown">
+      <ul class="navbar-nav ml-auto">
+      {
+        Navlinks.map((i) =>
+            /*--------For large screen---------- */
```
Comment (ashishnagpal2498) [inline src/components/layout/Navbar.jsx:27]: Remove this comment.

**Review by ashishnagpal2498 — CHANGES_REQUESTED** [review]

**Comment by nikita-1801** [comment]
Made the changes

**Pull request merged**